### PR TITLE
resolve review comments

### LINKED ISSUE #41: resolve code comments
- [x] write common associations in model

- [x]  remove response object from functions

- [x] add logger file

### PR DESCRIPTION
**Pull request by avinash-mane**

### CODE REVIEW

File: models/sequelize.js [hunk 1]
```diff
@@ -15,22 +18,43 @@ if (config.use_env_variable) {
...
-  Sequelize
-);
-db.projects = require("./projects.model")(sequelize, Sequelize);
-db.projects_repositories = require("./projects_repositories.model")(
-  sequelize,
-  Sequelize
-);
-db.users_projects = require("./users_projects.model")(sequelize, Sequelize);
-db.commits = require("./commits.model")(sequelize, Sequelize);
+// db.roles = require("./roles.model")(sequelize, Sequelize);
```
Comment (sunil4sonawane) [inline models/sequelize.js]: why commented code here?

File: src/pages/api/find-project.js [hunk 2]
```diff
@@ -18,6 +20,9 @@ const findProject = async (req, res) => {
     res.status(200).json(projectList);
   } catch (err) {
     Sentry.captureException(err);
+    logger.error("Error executing in while find projects");
```
Comment (sunil4sonawane) [inline src/pages/api/find-project.js]: grammar mistake

**Review by sunil4sonawane — CHANGES_REQUESTED** [review]
check comments

File: models/sequelize.js [hunk 3]
```diff
@@ -15,22 +18,43 @@ if (config.use_env_variable) {
...
+    db[model.name] = model;
+  });
+
+Object.keys(db).forEach((modelName) => {
+  if (db[modelName].associate) {
+    db[modelName].associate(db);
+  }
+});
+
 db.sequelize = sequelize;
```
Comment (sunil4sonawane) [inline models/sequelize.js]: why double initialisation here?

**Review by sunil4sonawane — CHANGES_REQUESTED** [review]
check comments

File: models/sequelize.js (hunk 3 above)
Comment (avinash-mane) [reply models/sequelize.js]: this are two different objects (sequelize, Sequelize)

Comment (sunil4sonawane) [reply models/sequelize.js]: same names makes conflict(Keep in mind when your code runs on windows case sensitivity will be simply ignored). Also what is the purpose of having these two objects?

**Review by sunil4sonawane — APPROVED** [review]

**Pull request merged**